.github/workflows/rlui-test.yml

```diff
@@ -19,7 +19,10 @@ on:
   pull_request:
     branches:
       - main
+  push:
+    branches:
       - staging
+      - feature/e2e-testing
```

Comment:
Do we need `feature/e2e-testing` as its not applicable to [GCP:staging ](https://github.com/GoogleCloudPlatform/rad-lab/tree/staging) ?

Reply:
I need it while testing. I will remove before merging.

Reply:
Sounds good.